src/lib/tailwind/settings.ts

```diff
@@ -1,8 +1,13 @@
 // Common Shared Settings and Constants
 
-module.exports = {
-	colorNames: ['primary', 'secondary', 'tertiary', 'success', 'warning', 'error', 'surface'],
-	colorShades: [50, 100, 200, 300, 400, 500, 600, 700, 800, 900],
+interface ColorPairing {
+	light: number;
+	dark: number;
+}
+
+export default {
+	colorNames: ['primary', 'secondary', 'tertiary', 'success', 'warning', 'error', 'surface'] as const,
```

Comment:
I've never seen ``as const`` used in Typescript, what does it do and is it necessary?

Reply:
[Matt Pocock's video explanation (2 mins)](https://www.youtube.com/watch?v=jxSh-RZTCws)
[Typescript release notes (2 mins)](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#const-assertions)
[Logrocket blog post (5 mins)](https://blog.logrocket.com/complete-guide-const-assertions-typescript/)

In short, it tells typescript that these values are hard-coded: they will never change. So the `typeof colorNames` is a string literal of `'primary' | 'secondary' | ...`. This gives us intellisense autocompletion for these color names